pkg/registry/apis/provisioning/jobs/pullrequest/preview.go

```diff
@@ -8,43 +8,43 @@ import (
 	"net/url"
 	"path"
 
-	"github.com/grafana/grafana-app-sdk/logging"
+	provisioning "github.com/grafana/grafana/pkg/apis/provisioning/v0alpha1"
 	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
-	"github.com/grafana/grafana/pkg/registry/apis/provisioning/repository"
+	"github.com/grafana/grafana/pkg/registry/apis/provisioning/resources"
 )
 
 // resourcePreview represents a resource that has changed in a pull request.
 type resourcePreview struct {
-	Filename              string
-	Path                  string
-	Action                string
-	Kind                  string
-	OriginalURL           string
-	OriginalScreenshotURL string
-	PreviewURL            string
-	PreviewScreenshotURL  string
+	Filename             string
+	Path                 string
+	Action               string
+	Kind                 string
+	TargetURL            string
+	TargetScreenshotURL  string
```

Comment:
could keep it as "Original" but it seems more clear that this is the place we will save